### ISSUE TITLE
Create Catergories for APIs

### ISSUE THREAD

**Issue by DeveloperAspire** [kaf-lamed-beyt/ng-apis#4]
I think we need to create categories for APIS

for example, Paystack falls under FinTech, while Glovo falls under food delivery.

This will help users navigate faster to the API that they need using the categories feature.

I can take on this. @kaf-lamed-beyt  You can assign this to me too.

**Comment by kaf-lamed-beyt** [comment]
Yeah... I also thought about this. I was just too lazy to start structuring the list. I'm also thinking about something... what if we leave the link that places them in an alphabetical order, do you think it'll affect the categories section you're planning to add?

**Comment by kaf-lamed-beyt** [comment]
Oh wait! After a quick thought, I think it'd affect it. 

But, if you can find a workaround. That'd be absolutely lovely! 💯

**Comment by DeveloperAspire** [comment]
I think it would aff

> Yeah... I also thought about this. I was just too lazy to start structuring the list. I'm also thinking about something... what if we leave the link that places them in an alphabetical order, do you think it'll affect the categories section you're planning to add?

Yeah, I think it will though.

**Comment by DeveloperAspire** [comment]
> Oh wait! After a quick thought, I think it'd affect it.
> 
> But, if you can find a workaround. That'd be absolutely lovely! 100

Yeah it would, I will see what I can do about !
